3rdparty/CMakeLists.txt

```diff
@@ -223,8 +223,13 @@ add_subdirectory(OpenAL EXCLUDE_FROM_ALL)
 set(FAUDIO_TARGET 3rdparty_dummy_lib)
 if(USE_FAUDIO)
 	if (USE_SYSTEM_FAUDIO)
-		find_package(FAudio)
-		set(FAUDIO_TARGET FAudio)
+		message(STATUS "RPCS3: Using system FAudio")
```

Comment:
This should be consistent with the other messages and say
`-- RPCS3: using shared FAudio`
and if we're adding a message for shared, we should add a message for the builtin branch as well.

Reply:
when you use status it adds the `--` to the start of the message, so the resulting message is actually `-- -- RPCS3:`

Reply:
Ah, I see. For consistency, probably in a follow up PR, the other messages should be changed to the STATUS mode.

Reply:
okay, sure :)

Reply:
Can you still add the message for the builtin branch

Reply:
sure :)